packages/docs/content/foundation/system-color-tokens.mdx

```diff
@@ -1,7 +1,9 @@
 ---
 title: System color tokens
 order: 40
-intro: System color tokens are the complete set of color tokens from which themes are built.
+intro: System color tokens are the entire set of color tokens from which our themes are built.
 ---
 
+However, when building your product, it's essential to use the theme color tokens that have been defined for your Brand's experience. Think of System color tokens like an entire box of crayons, our themes use specific colors from the box, not all of them.
```

Comment:
```suggestion
However, when building your product, it's essential to use the theme color tokens that have been defined for your brand's experience. Think of System color tokens like an entire box of crayons; our themes use specific colors from the box, not all of them.
```

This second sentence is technically two; otherwise, it's considered a comma splice. I replaced the comma with a semicolon.

The other edit I'm unsure about. Did you mean to capitalize _Brand_?